app/controllers/Application.scala

```diff
@@ -554,10 +562,10 @@ object Application extends Controller {
     }
   )
 
-  def getNotebook(name: String, path: String, format: String, dl:Boolean=false) = {
+  def getNotebook(name: String, path: String, format: String, dl: Boolean = false) = {
     try {
       Logger.debug(s"getNotebook: name is '$name', path is '$path' and format is '$format'")
-      val response = nbm.getNotebook(path).map { case (lastMod, name, data, path) =>
+      val response = nbm.getNotebook(path).map { case (lastMod, `name`, data, `path`) =>
```

Comment:
I wouldn't keep the backticks here too, and actually, since the variable aren't use we could simply use `_`